rust/src/delta.rs

```diff
@@ -207,6 +207,29 @@ pub struct DeltaTableMetaData {
     pub configuration: HashMap<String, String>,
 }
 
+impl DeltaTableMetaData {
+    fn new(
+        name: Option<String>, 
+        description: Option<String>, 
+        schema: Schema, 
+        partition: Vec<String>, 
+        config: HashMap<String, String>
+    ) -> Self {
+        //use inputs
+        
+        Self {
+            id: Uuid::new_v4().to_string(), //create a new GUID
```

Comment:
👍  
Delta uses `java.util.UUID.randomUUID().toString` to create the [GUID](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#change-metadata): https://github.com/delta-io/delta/blob/master/core/src/main/scala/org/apache/spark/sql/delta/actions/actions.scala#L350

Reply:
are you saying that this looks correct for generating a new guid?

Reply:
Yes, it could be helpful to add this reference in the code as a comment.